File: Cmentarz/Cmentarz/Models/Grobowiec.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cmentarz.Models
{
    [Table("Grobowce")]
    public class Grobowiec
    {
        [Key]
        public int IdGrobowiec { get; set; }
        [Required]
        public string IdWlasciciel { get; set; }
        [Required]
        public string Lokalizacja { get; set; }
        public decimal Cena { get; set; }
        public List<string>ListaOdwiedzajacy { get; set; }
        [Required]
        bool CzyZajety { get; set; }
    }
}

File: Cmentarz/CmentarzAngular/src/app/models/grobowiec.ts
import { Odwiedzajacy } from "./odwiedzajacy";
import { Zmarly } from "./zmarly";

export interface Grobowiec {
    idGrobowiec: number;
    lokalizacja:string;
    idWlasciciel:number;
    czyZajety:boolean;
    cena:number;
    sektor:string;
    ListaOdwiedzajacy?: Odwiedzajacy[];
    Zmarli?: Zmarly[];
  }